### ISSUE TITLE
Redstone torches placed on a block adjacent to an active motion sensor deactivate instantly

### ISSUE THREAD

**Issue by theothertored** [Better-than-Adventure/bta-issues#338]
**Describe the bug**
A motion sensor doesn't power adjacent blocks. Despite that, redstone torches placed on blocks adjacent to an active motion sensor deactivate:

![2024-08-25_13 28 53](https://github.com/user-attachments/assets/0fe6aafa-65dd-4b10-9778-63576b4a8c55)

The torch does not activate again until given a block update. After the torch is reactivated, it does not deactivate even when the motion sensor is activated.

**Expected behavior**
Placed torches shouldn't be affected by the motion sensor, since it doesn't power adjacent blocks.

**Version info:**
 - Version 7.2_01

**Comment by Bigjango13** [comment]
> A motion sensor doesn't power adjacent blocks.

Yes it does, one (very useful) quirk of motion sensors is that it doesn't update adjacent blocks, including redstone. This allows for any block to be BUDed.

**Comment by theothertored** [comment]
@Bigjango13 in that case, this issue is unnecessary and I'm closing it.

**Comment by theothertored** [comment]
Actually, I'm reopening, I'd like to know if this is intentional and/or what the devs think about it. If this is a mechanic people find useful, I don't wanna rain on their parade, but I'd like a comment.
